### PR TITLE
feat: fetch NFT metadata from Communities

### LINKED ISSUE #9941: Support fetching Community collectibles
Collectibles used for community access do not follow the standard `tokenURI` metadata format. That means OpenSea will return empty metadata and won't provide the link.
In such cases, we need to get the `tokenURI` by interacting with the contract (see https://docs.infura.io/infura/tutorials/ethereum/retrieve-and-display-erc-721-and-erc-1155-tokens). Prefer Pokt, use Infura as fallback.
Our `tokenURI` will contain some sort of identifier for the corresponding community. We must then fetch the metadata and the image from Waku. Gonna need some help with this last part @0x-r4bbit @endulab

### PR DESCRIPTION
**Pull request by dlipicar**
Closes https://github.com/status-im/status-desktop/issues/9941

In this PR, we attempt to fetch NFT metadata from a Community when:
- OpenSea does not return any metadata
- The NFT contract implements `IERC721Metadata`
- The `tokenURI` returns a string with format `compressedCommunityID/tokenID`

A wrapper for collectible fetching was added, which for the moment simply fetches from OpenSea.

Tested with a Community Token minted to my account (`0xa1E277eA6b97eFfc5b61B3BF5dE03F438981247E`)

image

image

### CODE REVIEW

File: node/status_node_services.go [hunk 1]
```diff
@@ -473,10 +473,17 @@ func (b *StatusNode) appmetricsService() common.StatusService {
 
 func (b *StatusNode) walletService(accountsDB *accounts.Database, accountsFeed *event.Feed, openseaAPIKey string) common.StatusService {
 	if b.walletSrvc == nil {
+		var extService *ext.Service
+		if b.WakuV2ExtService() != nil {
```
Comment (endulab) [inline node/status_node_services.go:477]: You check WakuV2ExtService twice

Comment (dlipicar) [reply node/status_node_services.go:477]: 🤦

Comment (dlipicar) [reply node/status_node_services.go:477]: BTW is this a proper approach? Not sure (no idea) what the deal is with WakuV1 and WakuV2

Comment (stefandunca) [reply node/status_node_services.go:477]: We don't use WakuV1 anymore, so I expect the latter check not to help.

**Review by stefandunca — APPROVED** [review]

**Review by endulab — APPROVED** [review]

**Pull request merged**